index.js

```diff
@@ -0,0 +1,13 @@
+const calculateSubTotals = (items) => {
+  const total = items.price * (1 - items.discount) * items.quantity
+
+  // eslint-disable-next-line no-cond-assign
+  if (items.total = parseInt(total * 100) / 100) {
+    return items
+  }
```

Comment:
Doing an assign in an if statement doesn't work. Basically it does the assignment, returns what it assigned so you will pretty much always run the if block. `if` is for when you only want the return to happen sometimes, here you always want to return the item after updating it.